### DISCUSSION TITLE
PluginsGLPI

### DISCUSSION THREAD

**Discussion by JonathanM-Giboire** [community/community#29900] [Other Feature Feedback, Questions, & Ideas]
Hello,
I have asked to modify a configuration file for the "Barcode" plugin and I would like to know within how many days I could have an answer from support,
Thanks for your feedback,

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/pluginsGLPI/barcode? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:
